fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/util/ResourceTypesCache.java

```diff
@@ -20,29 +20,29 @@
  * This class provides a static cache for FHIR Resource type names.
  */
 public class ResourceTypesCache {
-    private static final String CLASSNAME = ResourceTypesCache.class.getName(); 
```

Comment:
whitespace-only change, so might be fine to skip the copyright update.  if you happen to do another push, please update it.

Reply:
Fixed